concent_api/common/decorators.py

```diff
@@ -298,3 +312,30 @@ def wrapper(*args, **kwargs):
             )
             raise
     return wrapper
+
+
+def get_response_from_view_with_secure_retry_if_database_locked(
+    view,
+    request: WSGIRequest,
+    client_message: golem_messages.message,
+    client_public_key: bytes,
+    *args: Optional[Any],
+    **kwargs: Optional[Any],
+) -> HttpResponse:
+    sleep_time = settings.INITIAL_VIEW_RETRY_DELAY
+    for counter in range(settings.MAX_VIEW_RETRIES):
+        try:
+            response_from_view = view(request, client_message, client_public_key, *args, **kwargs)
+            return response_from_view
+        except DatabaseError as e:
```

Comment:
Is this retry for the lock or for the `get_or_create()`?
- If it's for the lock, then we don't need it after all (as we have determined when working on the code for workers).
- If it's for the creation, then we should only retry once. And only the creation operation should be retried.